pkg/templates/node_1.14.go

```diff
@@ -456,6 +456,7 @@ storage:
           rotateCertificates: true
           featureGates:
             NodeLease: false
+            CSIInlineVolume: true
```

Comment:
Is this the only change required on the kublet to make CSI work?

Reply:
This depends a lot on the used CSI driver. At least https://github.com/kubernetes-csi/csi-driver-image-populator worked with these settings.